tests/cli/test_cmds_status.py

```diff
@@ -1037,6 +1037,35 @@ def test_paasta_status_exception(system_paasta_config):
         )
 
 
+def test_format_kubernetes_replicaset_table_in_non_verbose(mock_kubernetes_status):
+    with mock.patch(
+        "paasta_tools.cli.cmds.status.format_kubernetes_replicaset_table", autospec=True
+    ) as mock_format_kubernetes_replicaset_table, mock.patch(
+        "paasta_tools.cli.cmds.status.bouncing_status_human", autospec=True
```

Comment:
this tests that `paasta status` can form the replicasets if it receives them from the API in non-verbose mode, but you should also have a test of the changes to paasta_tools/instance/kubernetes.py so that you can show that the API in non-verbose mode it still returns replicasets